golang/pkg/builder/container/container_builder.go

```diff
@@ -555,11 +557,13 @@ func getPortSet(natPortBindings map[nat.Port][]nat.PortBinding) nat.PortSet {
 }
 
 func logWrite(dc *DockerContainerBuilder, message string) {
+	if zerolog.GlobalLevel() == zerolog.InfoLevel {
+		return
+	}
 	if dc.logger != nil {
 		_, err := (*dc.logger).WriteString(message)
 		if err != nil {
-			//nolint
-			fmt.Printf("Failed to write log: %s", err.Error())
+			log.Error().Err(err).Msg("Failed to write log")
```

Comment:
No zerolog in builder please! builder attempts to be reusable by us and other regardless of **context**, adding zerolog to deps looks bad from lots of angles.